ui/src/app/package/package-search/package-search.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { PackageSearchComponent } from './package-search.component';
+
+describe('PackageSearchComponent', () => {
```

Comment:
you can place an `x` on the the describe `xdescribe` in order to skip that particular test.